config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```

Comment:
- [ ] I was unable to fetch the greetings from the frontend app due to blocking from CORS policy.

  ![eyeu](https://user-images.githubusercontent.com/36542986/205008878-52417f8c-e42a-497a-ae6c-6f287ebc5b4e.jpg)

  You can resolve this like so:
  - In your Gemfile, on line 37, uncomment the `rack-cors` gem
     
     ```
     gem "rack-cors"
     ```
   - Run `bundle install` to update gem config
   - In this `config/initializers/cors.rb` file, uncomment this highlighted code and change "example.com" to "*" to accept request from all origins.

      ```
      Rails.application.config.middleware.insert_before 0, Rack::Cors do
        allow do
          origins "*"
      
          resource "*",
            headers: :any,
            methods: [:get, :post, :put, :patch, :delete, :options, :head]
        end
      end
      ```

Reply:
Thank you @TedLivist